.changeset/tidy-jeans-give.md

```diff
@@ -0,0 +1,5 @@
+---
+'@capawesome/capacitor-age-signals': minor
```

Comment:
```suggestion
'@capawesome/capacitor-age-signals': patch
```

Since this plugin is still in development (v0.x.x) we need to increment the patch version for feature releases.